Fundamentals/Code/Source-Control/index-v10.md

```diff
@@ -0,0 +1,157 @@
+/umbraco/Data/*.sqlite.db-shm
+/umbraco/Data/*.sqlite.db-wal
+
+# Log files
+/umbraco/Logs/
+
+# Media files
+/wwwroot/media/
+
+```
+
+For most projects, this gitignore will be enough, and this article will not be an exhaustive list of how to version control Umbraco in all possible scenarios.
+
+However, we will go through the different files in order to give you an insight into the anatomy of an Umbraco website and therefore which parts to include in version control and which parts not to.
+
+![Typical set of Umbraco Project Folders](images/file-structure-v10.png)
+
+#### The Umbraco Folder
+
+The main folder where the Umbraco CMS resides in, is the `/umbraco` one inside your project.
+
+Most of the files and folders within the Umbraco folder, is already added to the default gitignore file. As most of the Umbraco CMS core files are embedded, the `/umbraco` folder contains primarily temporary files and log files, which are all added as Umbraco is installed.
+
+We recommend that you follow the structure of the default gitignore file, and do not include any temporary files, log files or cache files to git.
+
+Below are a set of general recommendations regarding the files within the `/umbraco` folder.
+
+- `/umbraco/data/TEMP` - This folder contains examine indexes, NuCache files, and so on, these are temporary and should not be committed.
+- `/umbraco/Logs` - Umbraco currently uses *Serilog*, and a file will be generated in this folder containing trace logs of your application, one JSON file for each day.
+- `/umbraco/mediacache` - *ImageSharp* ships with Umbraco and when an image is requested via the processor, e.g. to be resized or cropped, a cached version of the transformed image will be stored in this folder. (The [Imaging settings section](../../../Reference/Configuration/ImagingSettings/index.md) allows you to determine where this cache is stored)
+
+#### Umbraco Models Builder
+
+The strategy here will depend a little on which mode ['Umbraco Models Builder'](../../../Reference/Templating/Modelsbuilder/index.md) you have opted to work with.
+
+- **InMemoryAuto** (default), The models are generated in memory, no source control is required.
+- **SourceCodeManual** and **SourceCodeAuto**, The models are generated in the `/umbraco/models` folder of your project (or can be configured to be in a different folder or project), allowing you to track changes to the models in source/version control.
+
+#### Media
+
+The Media section of Umbraco (unless configured otherwise) stores files in the `/wwwroot/media` folder. These can be updated by editors, in the Umbraco backoffice, so generally speaking, you would not source control these files.
+
+These are by default ignored by git.
+
+#### Packages and Plugins
+
+The **App_Plugins** folder is the home for all third-party packages installed on your site.
+
+Depending on how you installed the plugin it will affect how you choose to version control a particular third party plugin:
```

Comment:
```suggestion
Depending on how you installed the plugin it will affect how you choose to version control a particular third-party plugin:
```
```suggestion
Depending on how you installed the plugin it will affect how you choose to version control a particular third party plugin:
```